src/ProgramRepresentation/PVAliasSet.cpp

```diff
@@ -0,0 +1,117 @@
+#include "ProgramRepresentation/PVAliasSet.h"
+
+PVAliasSet::PVAliasSet() {}
+
+bool PVAliasSet::contains(ProgramVariable programVar) {
+  for (ProgramVariable pv : programVariables) {
+    if (pv.getRawName() == programVar.getRawName()) {
+      return true;
+    }
+  }
+
+  return false;
+}
+
+bool PVAliasSet::containsByRawName(const std::string &rawName) {
+  for (ProgramVariable pv : programVariables) {
+    if (pv.getRawName() == rawName) {
+      return true;
+    }
+  }
+
+  return false;
+}
+
+bool PVAliasSet::containsByCleanedName(const std::string &rawName) {
+  for (ProgramVariable pv : programVariables) {
+    if (pv.getCleanedName() == rawName) {
```

Comment:
This looks wrong to me. Maybe the variable name is just wrong (should it be `cleanedName`?)